janus_server/src/binary_utils.rs

```diff
@@ -53,7 +54,13 @@ pub fn datastore<C: Clock>(
     }
 
     let conn_mgr = Manager::new(database_config, NoTls);
+    let timeout = Some(Duration::from_secs(db_config.connection_pool_timeouts_secs));
     let pool = Pool::builder(conn_mgr)
+        .timeouts(Timeouts {
```

Comment:
FYI we will also need to call `.runtime(Runtime::Tokio1)` on the builder. Currently this fails with the following error.

```
Error: couldn't connect to database

Caused by:
    0: failed to create database connection pool
    1: Error occurred while building the pool: NoRuntimeSpecified: Timeouts require a runtime
```